inject/src/main/java/io/micronaut/inject/visitor/VisitorContext.java

```diff
@@ -37,7 +40,10 @@
  * @since 1.0
  */
 public interface VisitorContext extends MutableConvertibleValues<Object>, ClassWriterOutputVisitor {
+
     String MICRONAUT_BASE_OPTION_NAME = "micronaut";
+    String MICRONAUT_PROJECT_DIR = "micronaut.project.dir";
```

Comment:
Should be declared as an annotation processor property. See `@SupportedOptions` on `BeanDefinitionInjectProcessor`. Also I think we should be consistent and name the property `micronaut.processing.project.dir`